server/src/main/java/org/apache/cassandra/sidecar/metrics/ServerMetrics.java

```diff
@@ -53,4 +53,9 @@ public interface ServerMetrics
      * @return metrics related to internal caches that are tracked.
      */
     CacheMetrics cacheMetrics();
+
+    /**
+     * @return metrics related to invocation of C* JMX operations
+     */
+    JmxOperationsMetrics jmxOperationsMetrics();
```

Comment:
Should these Jmx operations metrics be part of a different PR?

Reply:
This is the first Sidecar endpoint for a jmx operation, we need this, we will use the same for upcoming endpoints as well which invoke C* JMX operations. I understand there are existing calls as well in Sidecar invoking C* JMX, we will need to modify them to publish metrics. I prefer to have JmxOperationsMetrics accessible in JmxClient.java and record metrics there, that way each endpoint doesn't have to worry about publishing JMX calls metrics, instead JmxClient will take care of this for every JMX call being made, but I need to see the possibility of this.

On another thought, do we really need to track metrics of JMX calls? Does JVM (or something in Java world) provide this already? If so, we don't need this. 

Note: this is different from Sidecar endpoints invocation metrics, which are already provided by vertx.